AUTHORS.md

```diff
@@ -0,0 +1,4 @@
+Authors
+=======
+- [Antranig Basman](https://github.com/amb26)
+- [Leopold Burdyl](https://github.com/vilnius-leopold)
```

Comment:
Thanks, it is very important to me to give credit to the right people!

License text is another topic. The way I did it, is that the right way of doing this?